client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/choice-blueprint/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import Form from 'calypso/a8c-for-agencies/components/form';
+import './style.scss';
+
+type Props = {
+	onContinue: () => void;
+	onSkip: () => void;
+};
+
+const ChoiceBlueprint: React.FC< Props > = ( { onContinue, onSkip } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Form
+			className="choice-blueprint-form"
+			title={ translate( 'Grow your business with a free personalized blueprint' ) }
+			description={ translate(
+				'By answering a few simple questions, we’ll provide tips on maximizing your agency’s impact.'
```

Comment:
This is not an apostrophe sign:

<img width="757" alt="Screenshot 2025-02-03 at 1 24 34 PM" src="https://github.com/user-attachments/assets/4fb048bb-3651-4a0e-8fe1-857c0e3e946a" />

```suggestion
				"By answering a few simple questions, we'll provide tips on maximizing your agency's impact."
```

Reply:
Good catch. These are just copied from Figma!